src/components/calendar/calendar-renderer/simple-calendar.ts

```diff
@@ -0,0 +1,240 @@
+    SEPTEMBER = 'september',
+    OCTOBER = 'october',
+    NOVEMBER = 'november',
+    DECEMBER = 'december'
+}
+
+enum WeekdayNames {
+    SUNDAY = 'sunday',
+    MONDAY = 'monday',
+    TUESDAY = 'tuesday',
+    WEDNESDAY = 'wednesday',
+    THURSDAY = 'thursday',
+    FRIDAY = 'friday',
+    SATURDAY = 'saturday'
+}
+
+enum PickerMode {
+    DAY = 'day',
+    MONTH = 'month',
+    YEAR = 'year'
+}
+
+@WithRender
+@Component
+export class MSimpleCalendar extends MAbstractCalendarRenderer {
+
+    @Prop({ default: PickerMode.DAY })
+    initialPickerMode: PickerMode;
+
+    @Prop({ default: true })
+    showMonthBeforeAfter: boolean;
+
+    @Prop({
+        default: () => {
+            return Object.keys(MonthsNames).map((key: string) => {
+                return Vue.prototype.$i18n.translate(
+                    `${TRANSLATION_MONTHS}.${MonthsNames[key]}${TRANSLATION_SUFFIXE}`
+                );
+            });
+        }
+    })
+    monthsNames: string[];
+
+    @Prop({
+        default: () => {
+            return Object.keys(MonthsNames).map((key: string) => {
+                return Vue.prototype.$i18n.translate(
+                    `${TRANSLATION_MONTHS}.${MonthsNames[key]}`
+                );
+            });
+        }
+    })
+    monthsNamesLong: string[];
+
+    @Prop({
+        default: () => {
+            return Object.keys(WeekdayNames).map((key: string) => {
+                return Vue.prototype.$i18n.translate(
+                    `${TRANSLATION_WEEKDAYS}.${WeekdayNames[key]}${TRANSLATION_SUFFIXE}`
+                );
+            });
+        }
+    })
+    daysNames: string[];
+
+    previousMonthLabel: string = this.$i18n.translate('m-calendar:previous.month');
+    nextMonthLabel: string = this.$i18n.translate('m-calendar:next.month');
+    previousYearLabel: string = this.$i18n.translate('m-calendar:previous.year');
+    nextYearLabel: string = this.$i18n.translate('m-calendar:next.year');
+
+    id: string = `m-simple-calendar-${uuid.generate()}`;
+
+    private now: Date = new Date();
+    private currentPickerMode: PickerMode = this.initialPickerMode;
+
+    onMonthClick(): void {
+        this.currentPickerMode = PickerMode.MONTH;
+    }
+
+    onYearClick(): void {
+        this.currentPickerMode = PickerMode.YEAR;
+    }
+
+    onYearSelect(event: Event): void {
+        this.currentPickerMode = PickerMode.MONTH;
+        super.onYearSelect(event);
+    }
+
+    onMonthSelect(event: Event): void {
+        this.currentPickerMode = PickerMode.DAY;
+        super.onMonthSelect(event);
+    }
+
+    hideDay(isInCurrentMonth: boolean): boolean {
+        return !isInCurrentMonth && !this.showMonthBeforeAfter;
+    }
+
+
+    get currentlyDisplayedYear(): number {
+        const currentYear: YearState | undefined = this.calendar.years.find((year: YearState) => year.isCurrent);
+        if (currentYear) {
+            return currentYear.year;
+        }
+        return this.now.getFullYear();
```

Comment:
return currentYear ? currentYear.year : this.now.getFullYear();
Même chose pour fonctions currentlyDisplayedMonth et currentYear et currentMonth